lib/diagnostic.rb

```diff
@@ -31,31 +31,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{character} is really a Sith Lord.' # double quotes would be needed, but that seems to mess up the rest of this doc.
```

Comment:
I'll give you credit here because of your comment. Double quotes are necessary though, it would only mess up the document if you forgot to delete a hanging quote from somewhere else so try to debug it next time.